app/models/concerns/traverse_data_json.rb

```diff
@@ -4,7 +4,7 @@ module TraverseDataJson
   #used to take a hash, deeply search and flatten the results into a file array
   #block expects an array to be returned
   def find_file_fields_and_transform!(hash, files, &block)
-    hash.each do |key, value|
+    hash&.each do |key, value|
```

Comment:
maybe we should put a guard clause on these instead?

Reply:
```
def find_file_fields_and_transform!(hash, files, &block)
 return {} if hash.nil?

Reply:
sure